src/blk/kernel/KernelDevice.cc

```diff
@@ -94,8 +94,8 @@ int KernelDevice::_lock()
   int fd = fd_directs[WRITE_LIFE_NOT_SET];
   uint64_t nr_tries = 0;
   for (;;) {
-    struct flock fl = { F_WRLCK,
-                        SEEK_SET };
+    struct flock fl = { .l_type = F_WRLCK,
```

Comment:
> explicit assign to fields in struct

@wjwithagen 
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

and explain the reason why we need to have this change.

Reply:
might want to replace "bluestore" with "blk/kernel" in the title.